frontend/src/api/index.ts

```diff
@@ -45,3 +64,37 @@ export const fetchCourse = async (
     return null;
   }
 };
+
+export const searchCourses = async (
+  searchQuery: string,
+  minIndex: number = 0,
+  maxIndex: number = 10
+): Promise<ScheduleCourse[]> => {
+  const courses: ScheduleCourse[] = [];
+  const proxyurl = "https://cors-anywhere.herokuapp.com/";
```

Comment:
Just curious, why do we need this?

Reply:
Search needs to enable CORS, this is a temporary measure